test/e2e/framework/deployment/fixtures.go

```diff
@@ -84,26 +83,6 @@ func CreateDeployment(client clientset.Interface, replicas int32, podLabels map[
 	return deployment, nil
 }
 
-// GetPodsForDeployment gets pods for the given deployment
-func GetPodsForDeployment(client clientset.Interface, deployment *appsv1.Deployment) (*v1.PodList, error) {
```

Comment:
Question that comes to mind, when should try and rewrite a functions v. moving it out of the framework?
For example, this one, we could rewrite it to not use k8s.io/kubernetes/pkg/controller/deployment/util.

If we want to move fast and get the framework out, should we leave at least a TODO comment or something to rewrite it later?

Reply:
Good question. I've been basing it off how many callers there are and how many times we'd have to duplicate it. If I have to duplicate it into more than 3 places I think a rewrite is warranted. In this case we only had to duplicate it into two other packages so a copy into those packages was easier.

Reply:
Though I can see future tests relying on this functionality since it's so generic, maybe a rewrite here is worth doing

Reply:
Between a TODO for later or a rewrite now I think this is one thing we should keep in the framework in someway - seems like a useful function to have nearby.

Reply:
sgtm, I'll rewrite it now

Reply:
updated

Reply:
damn, that was fast for such a PR!